audoma/examples.py

```diff
@@ -49,7 +50,13 @@ def generate_value(self) -> float:
         """
         min_val = float(getattr(self.field, "min_value", 1) or 1)
         max_val = float(getattr(self.field, "max_value", 1000) or 1000)
-        return random.uniform(min_val, max_val)
+        max_digits = getattr(self.field, "max_digits", None)
```

Comment:
I see `max_digits` it's not being used here, so I presume it may be deleted.
It also may cause a bug;
```
serializers.DecimalField(max_digits=None, decimal_places=3, default=None)
```
Above field definition will still use random for examples.